### ISSUE TITLE
HTTP Req/Min incorrect

### ISSUE THREAD

**Issue by lukeescude** [Unitech/pm2#5557]
When running with pm2 + Express, there's a metric exposed for HTTP Req/Min.

I believe this is incorrect, and it should say Req/Sec - All of my testing seems to indicate it's actually measuring requests per second, instead of per minute.

**Comment by angelo0000** [comment]
I sent an email about this same issue.  Even when I do a pm2 monit on the server directly it shows req/min but there is no way its /min.

**Comment by rammanokar-plateron** [comment]
Yes , It is Request / Sec, Verified by comparing the pm2_http metrics with cloudwatch metrics.
